lib/carrierwave/mongoid.rb

```diff
@@ -89,18 +89,24 @@ def find_previous_model_for_#{column}
           end
         end
 
-        def serializable_hash(options=nil)
-          hash = {}
-
-          except = options && options[:except] && Array.wrap(options[:except]).map(&:to_s)
-          only   = options && options[:only]   && Array.wrap(options[:only]).map(&:to_s)
-
-          self.class.uploaders.each do |column, uploader|
-            if (!only && !except) || (only && only.include?(column.to_s)) || (except && !except.include?(column.to_s))
-              hash[column.to_s] = _mounter(column.to_sym).uploader.serializable_hash
+        def serializable_hash(options = nil)
+          super(options).tap do |result_hash|
```

Comment:
@duhast Won't you have to extend the mount_uploaders method to make it work with mongoid?